castai/add-castai-removal-disabled/03-mutated.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+assert:
+- patched01.yaml
+- patched02.yaml
```

Comment:
These are fine to leave, but see previous comment that you should be created resources without the annotation or that have a different value.

Reply:
Got it, I misunderstood how the testing structure worked, updated I believe it should be correct now.